includes/class-freemius.php

```diff
@@ -9841,6 +9841,25 @@ function get_addon_by_slug( $slug, $flush = false ) {
             return false;
         }
 
+        /**
+         * @author Leo Fajardo (@leorw)
+         * @since 2.2.5.1
+         *
+         * @return array
+         */
+        function _get_plans_and_pricing_by_addon_id() {

```

Comment:
@fajardoleo the name is confusing because it sounds like the method expects an addon ID. I think that it would be better to name it `_get_addons_plans_and_pricing_map_by_id()`